routes/HealthCheck.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable max-params */
```

Comment:
could possibly throw the services in an options object to get rid of this

Reply:
Good thought, I made the change